File: back/Application/DTOs/DashboardSessao.cs
namespace Application.DTOs;
public class DashboardSessaoDTO
{
    public int SessaoId { get; set; }

    public int TotalAlunos { get; set; }
    public int Avaliaram { get; set; }
    public int Pendentes { get; set; }
    public decimal MediaGeral { get; set; }
    public int TotalNotas { get; set; }
    
    public List<CriterioDashboardDTO> Criterios { get; set; } = [];
    public List<GrupoDashboardDTO> Grupos { get; set; } = [];
}

File: front/webav360/src/app/Models/Dashboard/DashboardSessao.ts
import { CriterioDashboard } from './CriterioDashboard';
import { GrupoDashboard } from './GrupoDashboard';

export interface DashboardSessao {
  sessaoId: number;

  totalAlunos: number;
  avaliaram: number;
  pendentes: number;
  mediaGeral: number,
  totalNotas: number,
  notaMax: number
  turmaCod: string,

  criterios: CriterioDashboard[];
  grupos: GrupoDashboard[];
}
